src/main/kotlin/net/atos/zac/app/zaak/converter/RestZaakOverzichtConverter.kt

```diff
@@ -55,14 +55,18 @@ class RestZaakOverzichtConverter @Inject constructor(
                 }
             },
             behandelaar = takeIf { zaakrechten.lezen }?.let {
-                zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)
-                    .map { userConverter.convertUserId(it.betrokkeneIdentificatie.identificatie) }
-                    .orElse(null)
+                zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)?.let { rolMedewerker ->
```

Comment:
this can be condensed right?
```
zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)
    ?.betrokkeneIdentificatie
    ?.identificatie
    ?.let(userConverter::convertUser)
```